src/styles/main-menu.css

```diff
@@ -43,7 +43,8 @@
   }
 }
 
-.main-menu-logo {
+.main-menu-logo,
+.main-menu-switcher {
```

Comment:
Actually I think that using a single class instead of main-menu-logo and main-menu-switcher would be better but I could not decide what would be a convenient name for that class.